models/keyword_test.go

```diff
@@ -69,65 +63,112 @@ func (s *KeywordDBTestSuite) TestSaveKeywordsWithValidParams() {
 }
 
 func (s *KeywordDBTestSuite) TestSaveKeywordsWithEmptyStringSlice() {
-	keywords := []string{""}
+	bulkData := []Keyword{
+		{Keyword: "", UserID: s.userID},
+	}
 
-	result, err := SaveKeywords(s.userID, keywords)
+	result, err := SaveKeywords(bulkData)
 
 	assert.Equal(s.T(), nil, err)
 	assert.Equal(s.T(), 1, len(result))
 	assert.Equal(s.T(), "", result[0].Keyword)
 }
 
 func (s *KeywordDBTestSuite) TestSaveKeywordsWithEmptySlice() {
-	keywords := []string{}
+	bulkData := []Keyword{}
```

Comment:
While plurals should be avoided, so is data 😆 `keywordList` would be better than `bulkData`.

Reply:
Fixed in 7f34c5bce857f061f6556f6256609313a5f02e05